packages/env/lib/download-source.js

```diff
@@ -132,11 +131,10 @@ async function downloadZipSource( source, { onProgress, spinner, debug } ) {
 	log( 'Downloading zip file.' );
 	const zipName = `${ source.path }.zip`;
 	const zipFile = fs.createWriteStream( zipName );
-	await finished(
-		requestProgress( request( source.url ) )
-			.on( 'progress', ( { percent } ) => onProgress( percent ) )
-			.pipe( zipFile )
-	);
+
+	const resStream = got.stream( source.url );
+	resStream.on( 'downloadProgress', ( x ) => onProgress( x.percent ) );
```

Comment:
What is `x`? Let's give it a meaningful name or use destructuring.

```suggestion
	resStream.on( 'downloadProgress', ( { percent } ) => onProgress( percent ) );
```